src/vaadin-message-list.d.ts

```diff
@@ -24,7 +25,7 @@ import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.
  * @mixes ThemableMixin
  * @mixes ElementMixin
  */
-declare class MessageListElement extends ThemableMixin(ElementMixin(HTMLElement)) {}
+declare class MessageListElement extends ThemableMixin(ElementMixin(HTMLElement)) { }
```

Comment:
nit: please remove extra whitespace.

Apparently the Git hook we have is not configured to pick up `.d.ts` files.
You can run formatting manually using `npx prettier src --write`.

Reply:
Done.